examples

```diff
@@ -1 +0,0 @@
-docs/source/examples
```

Comment:
do you know why this file was deleted?

Reply:
no idea how I screwed that up. I have restored it